### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by rachelmariewhite**

### CODE REVIEW

File: public/home.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      <!-- import the webpage's stylesheet -->
+      <link rel="stylesheet" href="./src/global.css">
+      <!-- import the webpage's javascript file -->
+      <script src="https://unpkg.com/ionicons@latest/dist/ionicons.js"></script>
+      <script src="https://unpkg.com/@ionic/core@latest/dist/ionic.js"></script>
+      <link href="https://unpkg.com/@ionic/core@latest/css/ionic.bundle.css" rel="stylesheet">
+      <link rel="stylesheet" type="text/css" href="style.css">
+   </head>
+   <body>
+      <ion-grid>
```
Comment (romulocintra) [inline public/home.html]: ion-grid is not used to wrap ion-app

**Review by romulocintra — CHANGES_REQUESTED** [review]

File: public/home.html (hunk 1 above)
Comment (rachelmariewhite) [reply public/home.html]: moved down after ion-app

**Comment by rachelmariewhite** [comment]
corrected ion-grid after ion-app

**Review by romulocintra — APPROVED** [review]
LGTM

**Pull request merged**